setup.cfg

```diff
@@ -28,7 +28,7 @@ classifiers =
 zip_safe = False
 use_2to3=False
 packages = find:
-python_requires = >=3.8,<3.10
+python_requires = >=3.8,<=3.11
```

Comment:
This specifier won't match 3.11.1 but `>=3.8,<3.12` should work as intended
```
>>> from packaging.specifiers import SpecifierSet
>>> from packaging.version import Version
>>> SpecifierSet(">=3.8,<=3.11").contains(Version("3.11.1"))
False
>>> SpecifierSet(">=3.8,<3.12").contains(Version("3.11.1"))
True
>>> SpecifierSet(">=3.8,<3.12").contains(Version("3.12"))
False
```

Reply:
yes, thank you for catching this. i'll make the change rn.